### ISSUE TITLE
Suggestion for Wizards: Commands with the players name, permission for command via Wizard

### ISSUE THREAD

**Issue by Fureniku** [CitizensDev/Citizens#160]
Two very simple suggestions here.

The first is for the wizard to be able to execute a command with the players name, for example setting a Permissions group: "/pex user %name% set Paladin" or whatever. That would add a whole level of new functionality to the Wizards...

Secondly is quite related: The ability to use a Wizard to do some sort of command without being able to do the command yourself. It seems sort of weird the way it is - why go to the Wizard when you can do the command yourself? I wasn't quite sure how it would work, but I think the easiest way is to use a permissions node per Wizards ID, such as citizens.wizard.7 giving you access to whatever wizard with ID 7 can do.
This means players can add themselves to a group as above, but without being able to change to any group for any player whenever they like.

Also these together would be great with Towny - players could add themselves to towns and so on :)

**Comment by fullwall** [comment]
Yeah... commands are tough mainly due to permissions. I think what you're suggesting here is permissions for each wizard? Hmm...
